packages/plugin-node-unhandled-rejection/unhandled-rejection.js

```diff
@@ -3,16 +3,19 @@ module.exports = {
   load: client => {
     if (!client._config.autoDetectErrors || !client._config.enabledErrorTypes.unhandledRejections) return
     _handler = err => {
-      const event = client.Event.create(err, false, {
+      // if we are in an async context, use the client from that context
+      const c = (client._clientContext && client._clientContext.getStore()) ? client._clientContext.getStore() : client
```

Comment:
allows us to send request context for unhandled rejections originating from a request